### PR TITLE
Create send_batch_data method

### LINKED ISSUE #72: Create multiple records are not implemented in CircuitPython
In a follow-up post with my PR (#70), the library should need to have the ability to create multiple records in one feed or multiple feeds, and I'm surprised that it doesn't have the implementation just yet. I'm open to ideas for the feature implementation.

### PR DESCRIPTION
**Pull request by rodrigoargumedo**
This would allow CircuitPython to send a batch array of data to be created from all available devices to the Adafruit.IO service. I will require further testing in all of my CircuitPython devices in the coming days and will mark this for review once it is complete.

Closes #72

Reference: https://io.adafruit.com/api/docs/?python#create-multiple-data-records

### CODE REVIEW

**Comment by rodrigoargumedo** [comment]
I'm not sure why the pre-commit hook is failing when they are all passing, but I'm looking into it why is it not passing.

**Comment by brentru** [comment]
@rodrigoargumedo  This is marked as a draft PR - is it ready for review yet? What's blocking it?

**Comment by rodrigoargumedo** [comment]
@brentru The only code block is the unexpected JSON parsing error when I tested it with my Matrix Portal. It sounded pretty odd, but I was expecting something a value would be sent to the service itself.

**Comment by rodrigoargumedo** [comment]
I may believe that I incorrectly put the POST method instead of directing the `send_data` method, which could've been expected.

**Comment by rodrigoargumedo** [comment]
I found the solution and getting some more tests readying up soon.

**Comment by rodrigoargumedo** [comment]
Now, I am ready for code review.

**Comment by brentru** [comment]
@rodrigoargumedo  Could you post an example `code.py` that you're using to test this with so I can see how this function would be implemented?

File: adafruit_io/adafruit_io.py [hunk 1]
```diff
@@ -565,6 +565,17 @@ def send_data(self, feed_key, data, metadata=None, precision=None):
         payload = self._create_data(data, metadata)
         self._post(path, payload)
 
+    def send_batch_data(self, feed, data_list):
+        """
+        Sends a batch array of data to a specified Adafruit IO feed
+        :param str feed_key: Adafruit IO feed key
+        :param list Data: Data list to send
```
Comment (brentru) [inline adafruit_io/adafruit_io.py:572]: The kwarg on L568 is called `data_list`, this line should be `:param list data_list`.

File: adafruit_io/adafruit_io.py [hunk 2]
```diff
@@ -565,6 +565,17 @@ def send_data(self, feed_key, data, metadata=None, precision=None):
         payload = self._create_data(data, metadata)
         self._post(path, payload)
 
+    def send_batch_data(self, feed, data_list):
+        """
+        Sends a batch array of data to a specified Adafruit IO feed
+        :param str feed_key: Adafruit IO feed key
```
Comment (brentru) [inline adafruit_io/adafruit_io.py:571]: The kwarg should be changed to `feed_key` on L568 to match the other method signatures.

**Review by brentru — CHANGES_REQUESTED** [review]

**Comment by rodrigoargumedo** [comment]
I'll do those changes tonight and push them. Thanks! 👍

**Review by brentru — APPROVED** [review]

**Comment by brentru** [comment]
Thanks for making these changes.

**Pull request merged**